packages/mobile/src/identity/privateHashing.ts

```diff
@@ -41,10 +41,13 @@ export function* fetchPhoneHashPrivate(e164Number: string) {
       Logger.error(`${TAG}@fetchPhoneHashPrivate`, 'ODIS insufficient balance', error)
       throw error
```

Comment:
I think we can remove this, cause `doFetchPhoneHashPrivate` would never throw this error.

Reply:
Good catch, I will delete this if clause!